pkg/controller/postgres/postgres_controller.go

```diff
@@ -142,7 +143,9 @@ func (r *ReconcilePostgres) Reconcile(request reconcile.Request) (_ reconcile.Re
 				return reconcile.Result{}, err
 			}
 		}
-		instance.SetFinalizers(nil)
+		if !suspendDBDeletion {
+			instance.SetFinalizers(nil)
+		}
```

Comment:
Ça fait planter les tests ici quand j'ai tenté `make unit-test` (j'ai inspecté pour avoir l'erreur, c'est `no kind is registered for the type v1alpha1.PostgresUserList in scheme "k8s.io/client-go/kubernetes/scheme/register.go:69"`).

Je pense qu'il faut rajouter ici : 

https://github.com/WTTJ/postgres-operator/blob/dec756ab99cc2348ce0a95cf85b2371ef7b1873c/pkg/controller/postgres/postgres_controller_test.go#L41

Que le schema mock accepte un type en plus : 

```golang
		sc.AddKnownTypes(v1alpha1.SchemeGroupVersion, &v1alpha1.PostgresUserList{})
```